.travis.yml

```diff
@@ -2,30 +2,20 @@ language: csharp
 
 mono: 5.12.0
 
-dotnet: 2.1.300
+dotnet: 3.1.300
+
+os:
+  - linux
+  - osx
+dist: xenial # Ubuntu 16
```

Comment:
changed from trusty due to https://travis-ci.community/t/not-able-to-install-net-core-3/5562